__tests__/index.ts

```diff
@@ -101,24 +100,31 @@ describe('Redirects', () => {
     })
 
     test('redirects when current path is different than given path', async () => {
+      serverMock('https://api.serlo.org/graphql', returnResponseJson(''))
+
+      /*
       mockFetch({
         'https://api.serlo.org/graphql': createJsonResponse({
           data: { uuid: { __typename: 'Article', alias: '/current-path' } },
         }),
       })
+      */
 
       const response = await handleUrl('https://en.serlo.org/path')
 
       expectToBeRedirectTo(response, 'https://en.serlo.org/current-path', 301)
     })
 
     test('no redirect when current path is different than given path and XMLHttpRequest', async () => {
+      serverMock('https://en.serlo.org/path', returnResponseJson(''))
+      /*
       mockFetch({
         'https://en.serlo.org/path': 'article content',
         'https://api.serlo.org/graphql': createJsonResponse({
           data: { uuid: { __typename: 'Article', alias: '/current-path' } },
         }),
       })
+      */
```

Comment:
Here you need two mocks for fetch. One for `https://en.serlo.org/path` and one for `https://api.serlo.org/graphql`. Note, that you need to return the right API response for the call to `https://api.serlo.org/graphql`.